Do not include GLES header when OpenGL headers are already included.

Taken from upstream:
https://hg.libsdl.org/SDL/rev/369b01006eb2

diff -r 4cbaffd0083b -r 369b01006eb2 src/video/SDL_video.c
--- a/src/video/SDL_video.c	Fri Oct 11 06:18:24 2019 +0200
+++ b/src/video/SDL_video.c	Sat Oct 12 18:47:56 2019 +0200
@@ -37,9 +37,9 @@
 #include "SDL_opengl.h"
 #endif /* SDL_VIDEO_OPENGL */
 
-#if SDL_VIDEO_OPENGL_ES
+#if SDL_VIDEO_OPENGL_ES && !SDL_VIDEO_OPENGL
 #include "SDL_opengles.h"
-#endif /* SDL_VIDEO_OPENGL_ES */
+#endif /* SDL_VIDEO_OPENGL_ES && !SDL_VIDEO_OPENGL */
 
 /* GL and GLES2 headers conflict on Linux 32 bits */
 #if SDL_VIDEO_OPENGL_ES2 && !SDL_VIDEO_OPENGL

